### ISSUE TITLE
How can I load my own dataset?

### ISSUE THREAD

**Issue by mahfuj9346449** [flyyufelix/cnn_finetune#10]
Suppose I have 20 categories images from 20 sub-directories. How can I load my own data ?

**Comment by jeremi** [comment]
check this issue: #4

**Comment by flyyufelix** [comment]
In short, you have to turn your image data into format compatible with Keras API. If you use python, that means writing a python script that iterates through your 20 directories and aggregates the image data and ground truth labels into numpy arrays (X_train, Y_train, X_valid, Y_valid respectively), where [X_train, Y_train] represents your training data and the corresponding labels, [X_valid, Y_valid] is your validation set for evaluating performance.
